scripts/install.sh

```diff
@@ -51,38 +51,65 @@ else
     echo -e "${ORANGE}Please look up how to install Python 3.9 or later for your specific operating system.${NC}"
     exit 1
 fi
-
 #
-# Install needed system packages
+# Get os package manager
 #
-# Run apt-get update
-echo -e "${GREEN}Updating apt-get cache...${NC}"
-sudo apt-get update || exit 1
+if [[ -f /etc/os-release ]]; then
+    source /etc/os-release
+    if [[ "$ID_LIKE" == *"debian"* ]]; then
+        pkg_manager="apt-get"
+        update_cmd="sudo $pkg_manager update"
+        install_cmd="sudo $pkg_manager install -y"
+        echo -e "${GREEN}Detected Debian-based system. Using apt-get package manager.${NC}"
+    elif [[ "$ID_LIKE" == *"arch"* ]]; then
+        pkg_manager="pacman"
+        update_cmd="sudo $pkg_manager -Sy"
+        install_cmd="sudo $pkg_manager -S --noconfirm"
+        echo -e "${GREEN}Detected Arch-based system. Using pacman package manager.${NC}"
+    else
+        echo -e "${ORANGE}Your operating system is not supported. Either try to install manually or reach out to spoolman github for support.${NC}"
+        exit 1
+    fi
+fi
 
-install_packages=0
+# Run pkg manager update
+echo -e "${GREEN}Updating $pkg_manager cache...${NC}"
+$update_cmd || exit 1
+packages=""
 if ! python3 -c 'import venv, ensurepip' &>/dev/null; then
     echo -e "${ORANGE}Python venv module is not accessible. Installing venv...${NC}"
-    install_packages=1
+    if [[ "$pkg_manager" == "apt-get" ]]; then
+        packages+="python3-venv"
+    elif [[ "$pkg_manager" == "pacman" ]]; then
+        packages+="python-virtualenv"
+    fi
 fi
 
 if ! command -v pip3 &>/dev/null; then
     echo -e "${ORANGE}Python pip is not installed. Installing pip...${NC}"
-    install_packages=1
+    if [[ "$pkg_manager" == "apt-get" ]]; then
+        packages+="python3-pip"
```

Comment:
You need to prepend all of these strings with a blank space, otherwise the final packages string will be like `"python3-venvpython3-piplibpq-dev"` which of course apt-get doesn't understand. Not sure how pacman handles this

Reply:
Of course, strange that i didn't trigger any issue with this one...